test/tools/unified-spec-runner/match.ts

```diff
@@ -675,11 +675,16 @@ export function filterExtraLogs(
 export function compareLogs(
   expected: ExpectedLogMessage[],
   actual: ExpectedLogMessage[],
-  entities: EntitiesMap
+  entities: EntitiesMap,
+  ignoreExtraMessages = false
 ): void {
-  expect(actual).to.have.lengthOf(expected.length);
+  if (!ignoreExtraMessages) {
```

Comment:
`ignoreExtraMessages` was in the schema previously, but now we have tests that use it so we ignore any extra messages after the expected ones from the unified suite.

Reply:
I've renamed the method as asked and added comments to the 2 properties directly from the spec so it's clear on what their purpose is. As for the filtering, I left it as it just before the call to `compareLogs` to leave the method signature there as is - seemed a bit clearer to understand.

Reply:
Looks good, thanks!